src/test/java/com/wikia/webdriver/testcases/visualeditor/VERecaptchaTests.java

```diff
@@ -22,7 +20,6 @@ public void setupd() {
 
   // Verify that second click on "Save page" button changes recaptcha.
   @Test(groups = {"VERecaptchaTests", "VEAddRecaptcha_001", "VEAnon"})
-  @RelatedIssue(issueID = "ZZZ-11954", comment = "test fails randomly")
```

Comment:
It was merged to master but not dev. It will be merged tomorrow by http://jenkins.wikia-prod:8080/view/CAT/view/selenium-tests/job/create-preview-test-run/ or we can merge it with this PR, doesn't matter.